assets/src/design-system/components/toggle/index.js

```diff
@@ -170,8 +170,8 @@ const ToggleContainer = styled.div(
   `
 );
 
-export const Toggle = (props) => (
-  <ToggleContainer>
+export const Toggle = ({ className, ...props }) => (
+  <ToggleContainer className={className}>
```

Comment:
allows us to use styled components on this component and have the classname apply to the outermost element

Reply:
Is the Toggle component used for the border-radius panel? Curious about the influence of this change on the PR.

Reply:
Good catch 😄 initially it was but I realized it didn't need to be. These changes shouldn't be included in this pr, so I'll remove them!